datacube_stats/main.py

```diff
@@ -321,18 +321,35 @@ def _log_config(self):
                   config['output_products'][0]['statistic'],
                   config['location'])
 
+    def _partially_applied_output_driver(self):
+        app_info = _get_app_metadata(self.config_file)
+
+        return partial(self.output_driver,
+                       output_path=self.location,
+                       app_info=app_info,
+                       storage=self.storage,
+                       global_attributes=self.global_attributes,
+                       var_attributes=self.var_attributes)
+
+    def execute_task(self, task):
+        """
+        Execute an individual task locally.
+        Intended to be used interactively rather than on a cluster.
+        """
+        try:
+            execute_task(task,
```

Comment:
Would be nice to check here to see if this task is going to use an XarrayReturning OutputDriver, and fail with a nice message if it's not. It would be annoying to accidentally load a long running job that isn't going to return you anything.

You would still get files on disk somewhere...

Reply:
Okay, user beware. 😄 🐉